api/swagger/swagger-v1.yaml

```diff
@@ -3789,6 +3789,53 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/coin_members_count_response'
+  /coins/volume-leaders:
+    get:
+      tags:
+        - coins
+      operationId: Get Volume Leaders
+      description: 'Gets top coin<>AUDIO trading addresses by volume'
+      parameters:
```

Comment:
this is fine, but personally i would have just done a single date param that would just find the volume b/w those dates. this is more expressive, but it led you to having to do some defensive programming (like max time range)